### PR TITLE
DSD-1205: expanding color theme object

### PR DESCRIPTION
**Pull request by bigfishdesign13**
Fixes JIRA ticket [DSD-1205](https://jira.nypl.org/browse/DSD-1205)

## This PR does the following:

- Adds the following color options to the color theme object: `ui.bg.page`, `ui.typography.inverse.body`, `ui.typography.inverse.heading`, `dark.ui.typography.inverse.body`, and `dark.ui.typography.inverse.heading`.

## How has this been tested?

- local Storybook

## Accessibility concerns or updates

- n/a

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] Review the Vercel preview deployment once it is ready.

### CODE REVIEW

**Review by EdwinGuzman — CHANGES_REQUESTED** [review]
The "Colors" storybook page should be updated to reflect the change. Right now, the color card for the updated values don't render correctly.

**Review by EdwinGuzman — APPROVED** [review]
Update is fine. Just note that the PR description has `heading.inverse` instead of `inverse.heading` (similar for the other values).

**Comment by bigfishdesign13** [comment]
@EdwinGuzman I initially added more contrast details for the typography colors, but then I got caught up in adding more info for other colors as well.  That being the case, this now needs one more review and approval.

File: src/components/StyleGuide/ColorCard.tsx [hunk 1]
```diff
@@ -1,32 +1,76 @@
-import { Box } from "@chakra-ui/react";
+import { Box, HStack } from "@chakra-ui/react";
 import * as React from "react";
 
 import Heading from "../Heading/Heading";
 import Icon from "../Icons/Icon";
-import SimpleGrid from "../Grid/SimpleGrid";
+// import SimpleGrid from "../Grid/SimpleGrid";
```
Comment (EdwinGuzman) [inline src/components/StyleGuide/ColorCard.tsx]: Remove this.

File: src/components/StyleGuide/ColorCard.tsx [hunk 2]
```diff
@@ -1,32 +1,76 @@
...
 import Table from "../Table/Table";
 import Text from "../Text/Text";
 import useNYPLTheme from "../../hooks/useNYPLTheme";
 
+// Small Color Swatch
+interface SmallColorSwatchProps {
+  /** The backgroundColor of the color swatch. */
+  backgroundColor: string;
+  border?: boolean;
+  darkBorder?: boolean;
```
Comment (EdwinGuzman) [inline src/components/StyleGuide/ColorCard.tsx:17]: These two should have comments.

File: src/components/StyleGuide/ColorCard.tsx [hunk 3]
```diff
@@ -44,19 +88,23 @@ export const DataTable = (props: React.PropsWithChildren<DataTableProps>) => {
...
+    // colors: { ui },
   } = useNYPLTheme();
-  const grayxxxxDark = ui.gray["xxxx-dark"];
-  const grayxxxDark = ui.gray["xxx-dark"];
-  const grayLightCool = ui.gray["light-cool"];
-  const grayMedium = ui.gray.medium;
+  // const grayxxxxDark = ui.gray["xxxx-dark"];
+  // const grayxxxDark = ui.gray["xxx-dark"];
+  // const grayLightCool = ui.gray["light-cool"];
+  // const grayMedium = ui.gray.medium;
```
Comment (EdwinGuzman) [inline src/components/StyleGuide/ColorCard.tsx]: Remove lines 101 to 107.

File: src/components/StyleGuide/ColorCard.tsx [hunk 4]
```diff
@@ -260,8 +419,9 @@ export const ColorCard = (props: React.PropsWithChildren<ColorCardProps>) => {
       paddingTop="s"
       border={border}
     >
-      <SimpleGrid columns={2}>
-        <Box>
+      {/* <SimpleGrid columns={2}> */}
```
Comment (EdwinGuzman) [inline src/components/StyleGuide/ColorCard.tsx]: Remove.

File: src/components/StyleGuide/ColorCard.tsx [hunk 5]
```diff
@@ -277,13 +437,18 @@ export const ColorCard = (props: React.PropsWithChildren<ColorCardProps>) => {
...
+          dataDarkHeadingColor={dataDarkHeadingColor}
+          dataDarkBodyColor={dataDarkBodyColor}
           dataWhiteColor={dataWhiteColor}
+          dataBrandPrimaryColor={dataBrandPrimaryColor}
+          dataBrandSecondaryColor={dataBrandSecondaryColor}
           textColor={textColor}
         />
-      </SimpleGrid>
+      </HStack>
+      {/* </SimpleGrid> */}
```
Comment (EdwinGuzman) [inline src/components/StyleGuide/ColorCard.tsx]: Remove.

**Review by EdwinGuzman — CHANGES_REQUESTED** [review]

**Review by EdwinGuzman — APPROVED** [review]

**Pull request merged**